scripts/tests/equipment/equipment_weapons.lua

```diff
@@ -0,0 +1,82 @@
+-----------------------------------
+-- Equipment Weapon TP Reset Tests
+-- Tests that TP is reset when equipping/unequipping weapons from various slots
+--
+-- Note: slotID vs equipSlotID distinction:
+--   - slotID: Inventory slot ID (0 = "unequip operation")
+--   - equipSlotID: Equipment slot being modified (SLOT_MAIN=0, SLOT_SUB=1, SLOT_RANGED=2, etc.)
+--
+-- Unequip flow in EquipItem():
+--   if (slotID == 0) → UnequipItem(equipSlotID) → TP reset handled in UnequipItem() line 2141
+-----------------------------------
+
+describe('Equipment Weapons - TP Reset on Unequip', function()
+    ---@type CClientEntityPair
+    local player
+
+    before_each(function()
+        -- Spawn NIN/10 with Dual Wield unlocked for full equipment testing
```

Comment:
The way `NIN/10` is written is very strange and "full equipment testing" is not really what this is doing.
The comment is largely pointless anyway with it being rather self explanatory